core/src/test/java/org/elasticsearch/search/suggest/phrase/DirectCandidateGeneratorTests.java

```diff
@@ -113,12 +114,13 @@ public void testFromXContent() throws IOException {
         for (int runs = 0; runs < NUMBER_OF_RUNS; runs++) {
             DirectCandidateGeneratorBuilder generator = randomCandidateGenerator();
             XContentBuilder builder = XContentFactory.contentBuilder(randomFrom(XContentType.values()));
+            generator.toXContent(builder, ToXContent.EMPTY_PARAMS);
+            XContentBuilder shuffled = shuffleXContent(builder, Collections.emptySet());
```

Comment:
I wasn't using varargs because I wanted to use `Set#contains()`, but now I see that most of the time we have empty sets, so I will change that. Not exactly sure what you mean by `generator = shuffleXContent(builder, emptySet)`, can you explain?